os/arch/arm/src/imxrt/chip/imxrt_usb.h

```diff
@@ -0,0 +1,757 @@
+#define IMXRT_USBDEV_ENDPTNAKEN_OFFSET       0x017c	/* Endpoint NAK Enable (device) */
+#define IMXRT_USBOTG_PORTSC1_OFFSET          0x0184	/* Port status/control 1 (both) */
+#define IMXRT_USBOTG_OTGSC_OFFSET            0x01a4	/* OTG status and control (otg) */
+#define IMXRT_USBOTG_USBMODE_OFFSET          0x01a8	/* USB device mode (both) */
+
+/* Device side naming of common register offsets */
+
+#define IMXRT_USBDEV_USBCMD_OFFSET           IMXRT_USBOTG_USBCMD_OFFSET
+#define IMXRT_USBDEV_USBSTS_OFFSET           IMXRT_USBOTG_USBSTS_OFFSET
+#define IMXRT_USBDEV_USBINTR_OFFSET          IMXRT_USBOTG_USBINTR_OFFSET
+#define IMXRT_USBDEV_FRINDEX_OFFSET          IMXRT_USBOTG_FRINDEX_OFFSET
+#define IMXRT_USBDEV_BURSTSIZE_OFFSET        IMXRT_USBOTG_BURSTSIZE_OFFSET
+#define IMXRT_USBDEV_BINTERVAL_OFFSET        IMXRT_USBOTG_BINTERVAL_OFFSET
+#define IMXRT_USBDEV_PORTSC1_OFFSET          IMXRT_USBOTG_OTGSC_OFFSET
+#define IMXRT_USBDEV_USBMODE_OFFSET          IMXRT_USBOTG_USBMODE_OFFSET
+
+/* Host side naming of common registers */
+
+#define IMXRT_USBHOST_USBCMD_OFFSET          IMXRT_USBOTG_USBCMD_OFFSET
+#define IMXRT_USBHOST_USBSTS_OFFSET          IMXRT_USBOTG_USBSTS_OFFSET
+#define IMXRT_USBHOST_USBINTR_OFFSET         IMXRT_USBOTG_USBINTR_OFFSET
+#define IMXRT_USBHOST_FRINDEX_OFFSET         IMXRT_USBOTG_FRINDEX_OFFSET
+#define IMXRT_USBHOST_BURSTSIZE_OFFSET       IMXRT_USBOTG_BURSTSIZE_OFFSET
+#define IMXRT_USBHOST_BINTERVAL_OFFSET       IMXRT_USBOTG_BINTERVAL_OFFSET
+#define IMXRT_USBHOST_PORTSC1_OFFSET         IMXRT_USBOTG_OTGSC_OFFSET
+#define IMXRT_USBHOST_USBMODE_OFFSET         IMXRT_USBOTG_USBMODE_OFFSET
+
+/* Device endpoint registers */
+
+#define IMXRT_USBDEV_ENDPTSETUPSTAT_OFFSET   0x01ac	/* Endpoint setup status */
+#define IMXRT_USBDEV_ENDPTPRIME_OFFSET       0x01b0	/* Endpoint initialization */
+#define IMXRT_USBDEV_ENDPTFLUSH_OFFSET       0x01b4	/* Endpoint de-initialization */
+#define IMXRT_USBDEV_ENDPTSTATUS_OFFSET      0x01b8	/* Endpoint status */
+#define IMXRT_USBDEV_ENDPTCOMPLETE_OFFSET    0x01bc	/* Endpoint complete */
+
+#define IMXRT_USBDEV_ENDPTCTRL_OFFSET(n)     (0x01c0 + ((n) << 2))
+#define IMXRT_USBDEV_ENDPTCTRL0_OFFSET       0x01c0	/* Endpoint control 0 */
+#define IMXRT_USBDEV_ENDPTCTRL1_OFFSET       0x01c4	/* Endpoint control 1 */
+#define IMXRT_USBDEV_ENDPTCTRL2_OFFSET       0x01c8	/* Endpoint control 2 */
+#define IMXRT_USBDEV_ENDPTCTRL3_OFFSET       0x01cc	/* Endpoint control 3 */
+#define IMXRT_USBDEV_ENDPTCTRL4_OFFSET       0x01d0	/* Endpoint control 4 */
+#define IMXRT_USBDEV_ENDPTCTRL5_OFFSET       0x01d4	/* Endpoint control 5 */
+#define IMXRT_USBDEV_ENDPTCTRL6_OFFSET       0x01d4	/* Endpoint control 6 */
+#define IMXRT_USBDEV_ENDPTCTRL7_OFFSET       0x01d4	/* Endpoint control 7 */
+
+/* USB0 register (virtual) addresses **********************************************************/
+
+/* Device/host capability registers */
+#define IMXRT_USBOTG_HCCR_BASE               (IMXRT_USB_BASE+IMXRT_USBOTG_HCCR_OFFSET)
```

Comment:
Please provide space befor and after '+'
like: (IMXRT_USB_BASE + IMXRT_USBOTG_HCCR_OFFSET)

Reply:
I fixed it